web/src/components/Feedback.tsx

```diff
@@ -48,9 +49,10 @@ type FeedbackProps = {
   onFeedbackChanged: (isPositiveFeedback: boolean | null) => void
   onSubmit: () => void
   sendingStatus: SendingStatusType
-  searchTerm?: string
+  noResults: boolean | undefined
+  searchTerm: string | undefined
```

Comment:
❓ Why did you switch from `searchTerm?: string` to `searchTerm: string | undefined`?

Reply:
I think using optional types are sometimes risking not passing anything even though something should be passed. Forcing devs to explicitly pass `undefined` or `null` might prevent bugs. Since `Feedback` is and will only be used once (by `FeedbackContainer`) and `searchTerm` is always passed, I figured this lowers (minimally) the risk of missing this in a future refactoring.

So basically I think the following makes sense as a rule of thumb:
- If the component is used in multiple places and most use cases have the same parameter, an optional property (perhaps with a default) make sense (e.g. an optional property `disabled` that is `false` for most usages)
- If the component is used in only one or two places and it is not clear what the default should be, its better to do it explicitly

Reply:
Cool, thanks for the explanation :)